code/modules/events/rogue_drones.dm

```diff
@@ -10,12 +10,7 @@
 		if(C.name == "carpspawn")
 			possible_spawns.Add(C)
 
-	//25% chance for this to be a false alarm
-	var/num
-	if(prob(25))
-		num = 0
-	else
-		num = rand(2,6)
+	var/num = rand(4,10)
```

Comment:
are you sure? Gotta remember these things aren't spawning in one place. They spawn outside the station in random spots around the hull, and we often have a shield generator running these days. 

There's a pretty good chance of not even finding all of them before they get recalled, and due to their predictability (ablative vest ftw) and the fact they rarely meet civilians, they're not a big threat.